providers/dns/gandi/gandi_mock_test.go

```diff
@@ -1,6 +1,6 @@
 package gandi
 
-// CleanUp Request->Response 1 (setZone)
+// CleanUp Request->Response 1 (setZone).
 const cleanup1RequestMock = `<?xml version="1.0"?>
```

Comment:
This might be just a note for the next refactoring oppurtunity:

The only information these comments add are the API endpoint (here: setZone). I'm in favor of getting rid of the comment and integrate the endpoint name in the constant name (`cleanupSetZoneRequestMock` or sth. similar).